test/integ/test_put_get_user_stage.py

```diff
@@ -60,6 +60,63 @@ def test_put_get_large_data_via_user_stage(
     )
 
 
+@pytest.mark.aws
+@pytest.mark.parametrize(
+    "from_path", [True, pytest.param(False, marks=pytest.mark.skipolddriver)]
+)
+def test_put_small_data_use_s3_regional_url(
+    is_public_test, tmpdir, conn_cnx, db_parameters, from_path
+):
+    """[s3] Puts Small Data via User Stage using regional url."""
+    if is_public_test:
+        pytest.skip(
+            "This test requires to change the internal parameter and requires account admin privileges"
+        )
```

Comment:
We should have a `mark` named `internal` that does this. I know that the code has both of these. I think I'd prefer to keep the `internal` tag instead of this if statement being copied around.